### ISSUE TITLE
Updating an existing cluster does not work.

### ISSUE THREAD

**Issue by RomainLasso** [DataThirstLtd/azure.databricks.cicd.tools#61]
Hi,
The call to the "New-DatabricksCluster"  doesn't seem to work when the cluster already exists. 

I have the following error message : "Missing required field: size"

I added both the UniqueNames and Update switch, but the payload sent to Rest API is invalid. The cluster parameters are not added to the payload.

The issue is probably at line 107 "New-DatabricksCluster.ps1". The "else" should be removed, and the cluster parameters must be added whether it's an add or edit command.

``` ps
 If ($UniqueNames)
    {
        $ClusterId = (Get-DatabricksClusters -Bearer $BearerToken -Region $Region | Where-Object {$_.cluster_name -eq $ClusterName})
        if ($ClusterId){
            if ($Update){
                $Mode = "edit"
                $Body['cluster_id'] = $ClusterId.cluster_id
                Write-Warning "Cluster already exists - it will be updated to this configuration"
            }
            //...
        }
    }
    else{ //Remove the else
        $ClusterArgs['SparkVersion'] = $SparkVersion
        $ClusterArgs['NodeType'] = $NodeType
        //...

        $Body += GetNewClusterCluster @ClusterArgs
    }

$BodyText = $Body | ConvertTo-Json -Depth 10

```

Best regards, 

Romain

**Comment by simondmorias** [comment]
Thanks you are right - I will have a fix shortly

**Comment by simondmorias** [comment]
Fixed and released in version 1.1.19 on PSGallery. Any problems please re-open - and thanks again for reporting.
